components/NavigationDrawer/src/MDCBottomDrawerViewController.m

```diff
@@ -72,15 +72,16 @@ - (UIModalPresentationStyle)modalPresentationStyle {
   return UIModalPresentationCustom;
 }
 
-- (UIScrollView *)trackingScrollView {
-  return self.transitionController.trackingScrollView;
-}
-
 - (void)setTrackingScrollView:(UIScrollView *)trackingScrollView {
+  _trackingScrollView = trackingScrollView;
+  if ([self.presentationController isKindOfClass:[MDCBottomDrawerPresentationController class]]) {
+    MDCBottomDrawerPresentationController *bottomDrawerPresentationController =
+        (MDCBottomDrawerPresentationController *)self.presentationController;
+    bottomDrawerPresentationController.trackingScrollView = trackingScrollView;
```

Comment:
Does this cause problems if that `trackingScrollView` was previously set to something else? Is there an invariant somewhere that we document that the two properties should always point to the same object?

Reply:
Is it possible to do the setting here only if trackingScrollView is still nil?

Reply:
I don't think I am full understanding the question here. `MDCBottomDrawerViewController` creates an `MDCBottomDrawerTransitionController` which then creates a `MDCBottomDrawerPresentationController`. Are we concerned about a client using `MDCBottomDrawerViewController` with a custom `presentationController`? Sorry guys, I am confused about the original question.